test/e2e/manifest_test.go

```diff
@@ -56,6 +56,13 @@ var _ = Describe("Podman manifest", func() {
 			session.WaitWithDefaultTimeout()
 			Expect(session).To(ExitWithError())
 
+			session = podmanTest.Podman([]string{"manifest", "push", "--all", "foo"})
+			session.WaitWithDefaultTimeout()
+			Expect(session).To(ExitWithError())
+			// Push should actually fail since its not valid registry
+			Expect(session.ErrorToString()).To(ContainSubstring("requested access to the resource is denied"))
+			Expect(session.OutputToString()).To(Not(ContainSubstring("accepts 2 arg(s), received 1")))
```

Comment:
this NOT check is pointless, you would need to match `ErrorToString` to string in order to actually check what you are trying to do.  General speaking given that you already check for the error message above you could just drop this check.